src/main/java/com/clouway/chita/HttpClient.java

```diff
@@ -94,7 +94,11 @@
 
         }
 
-        inputStream = conn.getInputStream();
+        try {
```

Comment:
I tried using *if*, but apparently there is no error stream and code goes directly to getInputStream and the bug would persist.